delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/routes/StoragesRoutes.scala

```diff
@@ -259,7 +259,7 @@ object StoragesRoutes {
       case IncorrectRev(_, _)                => StatusCodes.Conflict
       case WrappedProjectRejection(rej)      => rej.status
       case WrappedOrganizationRejection(rej) => rej.status
-      case StorageNotAccessible(_, _)        => StatusCodes.Forbidden
+      case StorageNotAccessible(_, _)        => StatusCodes.BadRequest
```

Comment:
Why? If it is not accessible it is as if the client wouldn't have permissions

Reply:
`StorageNotAccessible` is raised when volumes / bucket are invalid no ? It is not another rejection when there are no permissions ?

Reply:
Now it seems to me more like a NotFound.
But yeah, bad request is fine I guess